### DISCUSSION TITLE
Should we add support for devcontainers and codespace?

### DISCUSSION THREAD

**Discussion by vadim-su** [astral-sh/ruff#4676] [Ideas]
Now, if you use the codespace (Github function), it creates an unconfigured container for the project without rust and other auxiliary utilities installed. I assume a ready-made environment in devcontainer will reduce the threshold of entry.

What do you think about that?

**Comment by MichaReiser** [comment]
I would like that! Do you know what is necessary to set it up or do you want to give it a try?

**Reply by vadim-su** [reply]
Sent PR #4678 to review )
